src/locales/en.js

```diff
@@ -6,6 +6,7 @@ export default {
     'app.permissionDenied': 'your session expired',
     'app.usingMocks': 'App Builder is currently using mocks',
     'app.welcome': 'Welcome to Entando, {name}!',
+    'app.coreUnreachable': 'Entando core is unreachable',
```

Comment:
Please use a more user-oriented message :)

Reply:
Can you please suggest one? :)

Reply:
app.serverError was already fine

Reply:
any contraindication with `app.serverError`?